UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/Data/GasMix.cs

```diff
@@ -438,6 +438,17 @@ public void MergeGasMixes(List<PipeData> otherGas)
 			}
 		}
 
+		public GasSO GetBiggestGasSOInMix()
+		{
+			GasSO bigGas = null;
+			foreach(var gas in GasData.GasesArray)
+			{
+				if (bigGas != null && gas.Moles < bigGas.Moles) continue;
+				bigGas = gas.GasSO;
+			}
+			return bigGas;
```

Comment:
```suggestion

			if(GasData.GasesArray.Count == 0)
			{
				return null;
			}
			
			GasSO bigGas = GasData.GasesArray[0];
			foreach(var gas in GasData.GasesArray)
			{
				if (gas.Moles < bigGas.Moles) continue;
				bigGas = gas.GasSO;
			}
			return bigGas;
```

Actually this avoids the null check.